src/Angor/Client/Pages/Investor.razor

```diff
@@ -486,7 +524,9 @@
             {
                 TransactionId = investment.TransactionId,
                 AmountInvested = investment.TotalAmount,
-                InvestorNPub = investorNostrPrivateKey.PubKey.ToHex()[2..]
+                InvestorNPub = investorNostrPrivateKey.PubKey.ToHex()[2..],
```

Comment:
if you call the endpoint stats you get this result
`/api/v1/query/Angor/projects/{projectId}/stats`

```
{
  "investorCount": 7,
  "amountInvested": 1700000000,
  "amountSpentSoFarByFounder": 130000000,
  "amountInPenalties": 580000000,
  "countInPenalties": 4
}
```

Perhaps this is what you are looking for?